src/Infrastructure/Controller/Regulation/RegulationDetailController.php

```diff
@@ -69,6 +73,7 @@ public function __invoke(string $uuid): Response
             'isPermanent' => $regulationOrderRecord->getRegulationOrder()->isPermanent(),
             'measures' => $measures,
             'regulationOrderRecord' => $regulationOrderRecord,
+            'regulationOrderHistory' => $regulationOrderHistory,
```

Comment:
Vu qu'on récupère l'élément d'historique le + récent concernant l'arrêté, on pourrait nommer la variable en conséquence, par exemple :

```suggestion
            'latestHistory' => $latestHistory,
```